### PR TITLE
Add fuzzy search

### PR DESCRIPTION
**Pull request by adriencaccia**
## Description

Use [fuzzy](https://www.npmjs.com/package/fuzzy) search.
This will for example make such a search possible:
image

Issue: #147 

## Tests

- [x] All tests passed.

### CODE REVIEW

**Comment by adriencaccia** [comment]
hey @carloscuesta, I moved the `fuzzy` filtering to a separate utils file and tested it, but the coverage diff is still not high enough :cry:.
Do you have an idea on how to get it higher ?
Thanks !

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "chalk": "^3.0.0",
     "conf": "^6.1.0",
     "execa": "^4.0.0",
+    "fuzzy": "^0.1.3",
```
Comment (carloscuesta) [inline package.json]: Can we replace this dependency for something else that is more maintained ?

https://www.npmjs.com/package/fuse.js

File: src/commands/search/index.js [hunk 2]
```diff
@@ -1,16 +1,14 @@
 // @flow
+import fuzzy from 'fuzzy'
```
Comment (carloscuesta) [inline src/commands/search/index.js]: This import is unused 🤔

**Comment by adriencaccia** [comment]
Thanks for the review !
Sorry I didn't follow the contributing guidelines before submitting the PR, I thought that since #147 existed you were waiting for someone to create it :wink:.
I used fuse.js instead of fuzzy and cleaned the code a bit !

**Pull request merged**